src/state/opamSysInteract.ml

```diff
@@ -97,41 +97,37 @@ type families =
   | Suse
 
 (* System status *)
-let family =
-  let family = lazy (
-    match OpamSysPoll.os_family () with
-    | None ->
+let family ?env () =
```

Comment:
```suggestion
let family ~env () =
```
I’d rather have the new parameter as non-optional to avoid easy tricky bugs when used

Reply:
(The env parameter is also present at several other places and the code currently doesn’t compile with just this suggestion)